demo/angular-editable-text.js

```diff
@@ -42,49 +43,80 @@
 
           elem.addClass('gg-editable-text');
 
-          scope.$watch('isEditing', function (val, oldVal) {
+          scope.onInputClick = function() {
+            scope.isEditing = true;
+            if (attrs.hasOwnProperty('selectAll')) {
+              input.select();
+            }
+          };
+
+          $timeout(function() {
```

Comment:
It's fixing this issue: https://github.com/GabiGrin/angular-editable-text/issues/7